test/BaselineOutput/Common/Text/featurized.tsv

```diff
@@ -1,11 +1,11 @@
d.image:kissboti.jpg."	"Word.image:kissboti.jpg.|however,"	Word.however,	Word.however,|it	Word.it|currently	Word.currently	Word.currently|is	Word.is|missing	Word.missing	Word.missing|information	Word.information|on	Word.on|its	Word.its|copyright	Word.copyright|status.	Word.status.	Word.status.|wikipedia	Word.wikipedia|takes	Word.takes	Word.takes|copyright	Word.copyright|very	Word.very	Word.very|seriously.	Word.seriously.	Word.seriously.|it	Word.deleted|soon,	Word.soon,	Word.soon,|unless	Word.unless	Word.unless|we	Word.we|can	Word.can|determine	Word.determine	Word.determine|the	Word.the|license	Word.license	Word.license|and	Word.source|of	Word.the|image.	Word.image.	Word.image.|if	Word.know|this	Word.this|information,	Word.information,	Word.information,|then	Word.add|a	Word.a|copyright	Word.copyright|tag	Word.tag|to	Word.image|description	Word.description	Word.description|page.	Word.page.|if	Word.any|questions,	Word.questions,	Word.questions,|please	Word.please|feel	Word.to|ask	Word.ask	Word.ask|them	Word.them|at	Word.the|media	Word.media|copyright	Word.copyright|questions	Word.questions|page.	Word.page.|thanks	Word.thanks|again	Word.again|for	Word.your|cooperation.	Word.cooperation.	Word.thanx	Word.thanx|efe,	Word.efe,	Word.efe,|i	Word.i|noticed	Word.noticed	Word.noticed|you	Word.you|remove	Word.remove|800	Word.800	Word.800|bytes	Word.bytes	Word.bytes|of	Word.of|info	Word.info|on	Word.my|watchlist	Word.watchlist	Word.watchlist|so	Word.so|i	Word.i|went	Word.went	Word.went|into	Word.into|red	Word.red	Word.red|alert	Word.alert	Word.alert|but	Word.good|call.	Word.call.	Word.==|woah!	Word.woah!	Word.woah!|==	Word.==|as	Word.as|someone	Word.someone|who'd	Word.who'd	Word.who'd|been	Word.been|the	Word.the|victim	Word.victim	Word.victim|of	Word.of|his	Word.his	Word.his|power	Word.power	Word.power|abuse,	Word.abuse,	Word.abuse,|this	Word.this|*really*	Word.*really*	Word.*really*|came	Word.came|as	Word.a|surprise	Word.surprise	Word.surprise|to	Word.me|when	Word.when|someone	Word.someone|e-mailed	Word.e-mailed	Word.e-mailed|this	Word.this|info	Word.info|to	Word.this|morning!	Word.morning!	Word.morning!|sorry	Word.sorry|he	Word.he|couldn't	Word.couldn't	Word.couldn't|be	Word.be|more	Word.more|adult	Word.adult	Word.adult|with	Word.with|his	Word.his|admin	Word.admin|powers,	Word.powers,	Word.powers,|but	Word.but|as	Word.as|stan	Word.stan	Word.stan|lee	Word.lee	Word.lee|said	Word.said|over	Word.over|four	Word.four|decades	Word.decades	Word.decades|ago,	Word.ago,	Word.ago,|with	Word.with|great	Word.great	Word.great|power	Word.power|comes	Word.comes|great	Word.great|responsibility.	Word.responsibility.	Word.responsibility.|of	Word.of|course,	Word.course,	Word.course,|the	Word.the|big	Word.big	Word.big|question	Word.question	Word.question|now	Word.now|is	Word.is|who	Word.who|matthew	Word.matthew	Word.matthew|fenton	Word.fenton	Word.fenton|will	Word.will|run	Word.run	Word.run|and	Word.and|hide	Word.hide	Word.hide|behind	Word.behind	Word.behind|when	Word.when|he	Word
```

Comment:
Bigrams are in the output now, which is good. For example, `Word.generally|accepted`. Tri-charactergrams are present.

The change from unigrams to bigrams appears to be working.